rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.create/20_synthetic_source.yml

```diff
@@ -1214,3 +1224,231 @@ fallback synthetic_source for text field:
       hits.hits.0._source:
         text: [ "world", "hello", "world" ]
 
+---
```

Comment:
Should we also have a reindex test now that we have `copy_to` working in synthetic source?

Reply:
I'll add it in a separate PR.